From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jake Bailey <5341706+jakebailey@users.noreply.github.com>
Date: Sun, 9 Oct 2022 13:14:58 -0700
Subject: [PATCH] Rename Gulpfile to Herebyfile for improved git diff

The next commit will switch the build system, but git doesn't detect
that the two config files are related.
---
 Gulpfile.mjs => Herebyfile.mjs | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 rename Gulpfile.mjs => Herebyfile.mjs (100%)

diff --git a/Gulpfile.mjs b/Herebyfile.mjs
similarity index 100%
rename from Gulpfile.mjs
rename to Herebyfile.mjs
-- 
2.38.0

